sphinxcontrib/inmanta/api.py

```diff
@@ -415,13 +440,18 @@ def run(self, module_repo: Optional[str], module_name: str, extra_modules: Seque
             raise Exception(f"Could not find module {module_name}.")
         mod, submodules = mod_data
 
+        module_filter = self.get_module_filter(None if not module_name else os.path.join(module_repo, module_name))
```

Comment:
Isn't it like this?
```suggestion
        module_filter = self.get_module_filter(None if not module_repo else os.path.join(module_repo, module_name))
```
Or are you trying to cover the case where module_name equals `""`? Still the first case should be covered

Reply:
Besides, why not rely on the module object?

Reply:
To keep it simple and predictable: in case of a V2 module that is checked out as source and installed, we want the to use the source to find the file.  (I don't even know if the pyproject.toml will be present or where it will be if the package is installed)